s3torchbenchmarking/utils/upload_colated_results_to_s3.py

```diff
@@ -0,0 +1,50 @@
+import os
+import boto3
+from botocore.exceptions import ClientError
+import sys
+
+s3_client = boto3.client("s3")
+
+
+def upload_file_to_s3(local_file_path: str, bucket_name: str, s3_file_key: str) -> None:
```

Comment:
Should we use something similar to `aws s3 sync` to avoid going through the files one by one?